app/views/casework/crime_applications/sections/_properties.html.erb

```diff
@@ -0,0 +1,141 @@
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:bedrooms, scope: 'properties') %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= property.bedrooms %>
+          </dd>
+        </div>
+        <% end %>
+        <% if property.property_type == PropertiesPresenter::LAND %>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:size_in_acres, scope: 'properties') %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= property.size_in_acres %>
+          </dd>
+        </div>
+        <% end %>
+        <% if property.property_type == PropertiesPresenter::LAND || property.property_type == PropertiesPresenter::COMMERCIAL %>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:usage, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= property.usage %>
+          </dd>
+        </div>
+        <% end %>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:value, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= number_to_currency(property.value / 100, unit: '£', separator: '.', delimiter: ',') %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:outstanding_mortgage, scope: 'properties') %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= number_to_currency(property.outstanding_mortgage / 100, unit: '£', separator: '.', delimiter: ',') %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:percentage_applicant_owned, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= number_to_percentage(property.percentage_applicant_owned, precision: 2) %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:percentage_partner_owned, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= number_to_percentage(property.percentage_partner_owned, precision: 2) %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:is_home_address, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= t(property.is_home_address, scope: 'values') %>
+          </dd>
+        </div>
```

Comment:
I believe these are partner fields. These fields should be hidden if it wasn't asked in apply